src/sentry/deletions/defaults/group.py

```diff
@@ -128,6 +131,9 @@ def delete_bulk(self, instance_list):
 
         group_ids = [group.id for group in instance_list]
 
+        # Tell seer to delete grouping records with these group hashes
+        call_delete_seer_grouping_records_by_hash(group_ids)
```

Comment:
should we add an option for this too?

Reply:
I have the feature check in the function itself since we needed to extract the project and it looked a bit messy

Reply:
ah gotcha 👍🏼